### PR TITLE
v2.6.0: Add-colours-menulog

### PR DESCRIPTION
**Pull request by smilburn1990**
###
- Added new colours for the `ml` theme

### CODE REVIEW

File: src/scss/index.scss [hunk 1]
```diff
@@ -85,9 +85,11 @@ $color-bg--accept             : $green--offWhite;
 $color-bg--notification       : $yellow--light;
 $color-bg--info               : $blue--offWhite;
 $color-bg--error              : $red--offWhite;
+$color-bg--default            : $blue;
```
Comment (ashleynolan) [inline src/scss/index.scss]: What's the intended difference between this and `$color-bg`? (as I'd expect that to be the `default`)

File: src/scss/index.scss [hunk 2]
```diff
@@ -85,9 +85,11 @@ $color-bg--accept             : $green--offWhite;
...
 
 $color-border                 : $grey--lightest;
 $color-border--interactive    : $grey--lighter;
+$color-border--default        : $blue;
```
Comment (ashleynolan) [inline src/scss/index.scss]: Same here – where will this be used (as may be able to name it a bit better so it's more obvious the difference from `$color-border`

File: src/scss/index.scss (hunk 1 above)
Comment (smilburn1990) [reply src/scss/index.scss]: Well i need it to be blue but we don't have one. Open to naming suggestions... @ashleynolan

File: src/scss/index.scss (hunk 2 above)
Comment (smilburn1990) [reply src/scss/index.scss]: Again with this one @ashleynolan

File: src/scss/index.scss [hunk 3]
```diff
@@ -110,6 +112,13 @@ $color-selection-bg           : inherit;
 // <hr> border color
 $hr-color                     : $grey--lightest;
 
+// Alert messages
+$color-alert-message          : $blue;
+
+// Text descriptions
+$color-text-description       : $blue;
```
Comment (ashleynolan) [inline src/scss/index.scss]: I'd move this to be next to the other `$color-text--` definitions on lines 74-78 and rename it to be `$color-text--description`

File: src/scss/index.scss (hunk 1 above)
Comment (ashleynolan) [reply src/scss/index.scss]: Have you got a design for where this is used? (as may be able to link it to the component type)

Comment (smilburn1990) [reply src/scss/index.scss]: Not the design but it is for this class...

`.c-basketTray`

@ashleynolan

File: src/scss/index.scss (hunk 2 above)
Comment (smilburn1990) [reply src/scss/index.scss]: And this has this class...

`c-scheduleTypeSwitcher-switch--active`

@ashleynolan

**Pull request closed without merging**